### DISCUSSION TITLE
Named Destination

### DISCUSSION THREAD

**Discussion by sanksara** [pikepdf/pikepdf#557] [Q&A] [answered]
I have been trying to create named destination to specific location on a page using pike pdf. Though named destinations are created, but they are not able to point to the specific x, y coordinates. they are able to move to the correct page of the named destination.

Request to help on how to zoom and point to a specific location on a page.

**Comment by jbarlow83** [comment]
You can create any PDF data structure you want using pikepdf. There are some structures where pikepdf has models to support manipulating them, but nothing prevents you from creating others.

The reference manual, which you can download for free, describes how to create a named destination -- although this might require a Go To action.

**Reply by sanksara** [reply]
Thanks .. could you help me to direct to the reference of how to add or manipulate named destination

**Reply by jbarlow83** [reply, marked as answer]
It's section 12.3.2.3 of the PDF 1.7 reference manual, which is widely available and free to download.
